### DISCUSSION TITLE
make_tiled_mma  Layout

### DISCUSSION THREAD

**Discussion by 18liumin** [NVIDIA/cutlass#1786] [Q&A]
Why Layout<Shape<_1, _2, _1>>{} has no effect ? 

auto tm = make_tiled_mma(SM80_16x8x16_F16F16F16F16_TN{},
                            Layout<Shape<_2,_2,_1>>{} 
                             );

auto tm = make_tiled_mma(SM80_16x8x16_F16F16F16F16_TN{},
                            Layout<Shape<_2,_2,_1>>{}，  
                            Layout<Shape<_1, _2, _1>>{}  
                             );
have the same structure 
  print_latex(tm);
  print(tm.get_thr_layout_vmnk()), print("\n");

**Comment by ccecka** [comment]
https://github.com/NVIDIA/cutlass/discussions/1345
